src/main/java/game/domain/Winners.java

```diff
@@ -0,0 +1,32 @@
+package game.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Winners {
+
+    private Winners() {
+    }
+
+    public static Cars findWinners(final Cars cars) {
+        return findWinners(cars, getMaxPosition(cars));
+    }
+
+    private static Cars findWinners(final Cars cars, final int maxPosition) {
+        List<Car> winners = new ArrayList<>();
+        cars.getCarList().stream()
+                .filter(car -> car.hasSamePosition(maxPosition))
+                .forEach(winners::add);
+        return new Cars(winners);
+    }
+
+    private static int getMaxPosition(final Cars cars) {
+        List<Position> positionList = cars.getCarList().stream()
```

Comment:
`Comparable` 을 구현한 `Position` 을 활용해서 값을 구해봐도 좋겠네요! :)

```suggestion
Position maxPosition = cars.getList().stream()
    .map(Car::getPosition)
    .max(Position::compareTo)
    .orElse(new Position());
```